lectureDL.py

```diff
@@ -527,17 +546,13 @@ def download_lectures_for_subject(driver, subject, current_year, week_day,
 
     # print status
     print("Building list of lectures...")
-    # scroll_wrapper = driver.find_elements
-    # driver.execute_script("window.scrollTo(0, document.body.scrollHeight);")
```

Comment:
Same thing here, I know it hurts to leave bad stuff behind, but you've gotta in order to keep your commits focused. In order to not forget to fix something up, you can open **another** branch and make these kinds of changes in that one. This was much easier in mercurial though, and rebasing was a breeze.